programs/Makefile

```diff
@@ -371,12 +359,11 @@ clean:
 ifndef WINDOWS
 	rm -f $(APPS)
 	-rm -f ssl/ssl_pthread_server$(EXEXT)
-	rm -f $(EXTRA_GENERATED)
 	-rm -f test/cpp_dummy_build$(EXEXT)
 else
 	if exist *.o del /Q /F *.o
 	if exist *.exe del /Q /F *.exe
-	del /S /Q /F $(EXTRA_GENERATED)
+	del /S /Q /F
```

Comment:
This line needs to be removed altogether.

Reply:
oh these are flags huh! never been on a DOS or NT CLI. I will get this fixed.